[core]/es_extended/client/main.lua

```diff
@@ -670,6 +670,28 @@ AddEventHandler("esx:killPlayer", function()
 	SetEntityHealth(ESX.PlayerData.ped, 0)
 end)
 
+RegisterNetEvent("esx:repairPedVehicle")
+AddEventHandler("esx:repairPedVehicle", function()
+	local GetVehiclePedIsIn = GetVehiclePedIsIn
+
+	ESX.TriggerServerCallback("esx:isUserAdmin", function(admin)
+		if not admin then
+			return
+		end
+		local ped = ESX.PlayerData.ped
+		if IsPedInAnyVehicle(ped, false) then
```

Comment:
If player not in car use case? Please handle if vehicle nil or 0 players is not a car, and add warning message.

Reply:
It's fixed, let me know if there's anything else :)